build.sbt

```diff
@@ -470,7 +471,12 @@ lazy val tests = crossProject(JSPlatform, JVMPlatform, NativePlatform)
     ),
     buildInfoPackage := "scala.meta.tests",
     libraryDependencies += "com.lihaoyi" %%% "fansi" % "0.2.5" % "test",
-    libraryDependencies += "org.scalatest" %%% "scalatest" % "3.2.0-SNAP10" % "test"
+    libraryDependencies += "org.scalatest" %%% "scalatest" % "3.2.0-SNAP10" % "test",
+    testOptions.in(Test) += Tests.Argument("-l", "org.scalatest.tags.Slow"),
```

Comment:
I love this change ❤️ Been meaning to come up with a solution for this problem for a long time. Thank you @laughedelic for the SO answer!